02-Appetisers/Barotropic_Streamfunction.ipynb

```diff
@@ -2,12 +2,20 @@
  "cells": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.    psi_mean.plot.contourf(
</pre><p>A single-sided colormap would make this clearer. There is nothing particularly special about the streamfunction equalling zero.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/COSIMA/cosima-recipes/pull/617/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>